doc/SRS.md

```diff
@@ -0,0 +1,174 @@
+Additional features might include the function to add and remove individual passwords as well as the change of settings.
+
+### 3.2 Usability
+We plan on designing any interface as simple and intuitive as possible while still providing maximum functionality. Of course there will be documents available explaining the usage, but we aim at making them redundant.
+#### 3.2.1 No training time needed
+Our goal is that any user can use our application without any training.
+#### 3.2.2 Natural and easy workflow
+We want to build an easy to use application that isn't overloaded with features.
+
+### 3.3 Reliability
+#### 3.3.1 Availability
+While we cannot 100% guarantee it, we want to ensure that the server is available at least 95% of the time, equivalent to approx. 1 hour of downtime per day.
+The time to repair bugs etc. should be as low as possible.
+#### 3.3.2 Defect Rate and Security
+Data should be stored in the most secure way possible. The data transfer is to be made via a secure connection.
+
+### 3.4 Performance
+#### 3.4.1 Response time
+Should be as low as possible. Maximum response time is 5 seconds.
+#### 3.4.2 Capacity
+The system should be able to manage multiple requests per second without noticeable latency. The system is to be build on a scalable basis.
+#### 3.4.3 Connection bandwidth
+The size of data received through one request should be as low as possible, e.g. sending duplicates of data should be avoided.
+
+### 3.5 Supportability
+#### 3.5.1 Coding standards
+In order to maintain supportability and readability of our code, we will adopt the clean code standard, common naming conventions uniform formatting and best practices throughout the project.
+#### 3.5.2 Maintenance Utilities
+In order to test and integrate newer versions of the application, a continuous integration service is required.
+
+### 3.6 Design Constraints
+Our goal is to provide a modern design in both code and application.  
+Server-side programs ought to be compiled using Golang. A REST-API will be used to communicate between client and server.  
+The dashboard will be implemented using a MVC architecture.  
+The chrome-plugin and the android autofill service will be implemented using their respective technologies and languages.  
+#### 3.6.1 Development tools
+- Version control system: Git (Github)
+- Backend development: JetBrains GoLand
+- Frontend development: tbd
+- Project planning tool: JetBrains YouTrack
+- Build management: tbd
+- CI: tbd
+
+### 3.7 On-line User Documentation and Help System Requirements
+As stated our goal is to make our application as intuitive as possible. However, we will provide a FAQ and documentation. This will be especially helpful for users who want to know how our application works in depth.
+
+### 3.8 Purchased Components
+A server hosted at [zap-hosting.com](https://zap-hosting.com/de/) is used to run the frontend application as well as the backend server.  
```

Comment:
maybe restructure this section
- domain at alldomains.hosting
- two servers hosted at zap
- we mentioned a KeyCloud@zeekay.dev mail already, mail server hosted at netcup

#forthesakeofdetails